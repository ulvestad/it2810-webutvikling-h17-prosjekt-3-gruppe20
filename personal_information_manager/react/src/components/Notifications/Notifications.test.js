import React from 'react';
import sinon from 'sinon';
import renderer from 'react-test-renderer';
import { shallow, mount } from 'enzyme';
import { expect } from 'chai';

import Notifications from './Notifications';

const jestExpect = global.expect;

it('renders without crashing', () => {
  shallow(<Notifications />);
});

it('renders correctly', () => {
  const tree = renderer.create(<Notifications />).toJSON();
  jestExpect(tree).toMatchSnapshot();
});

/* Wont work unitl localStorage uses getItem
it('gets data from localStorage when constructed', () => {
  const wrapper = shallow(<Notifications />);

  expect(localStorage.getItem.called).to.equal(true);
});
*/
