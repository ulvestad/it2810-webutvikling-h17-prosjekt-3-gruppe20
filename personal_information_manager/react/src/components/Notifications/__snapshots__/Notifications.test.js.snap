// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    className="bg-primary text-white"
    id="page-top"
  >
    <div
      className="container text-center"
    >
      <h2>
        Welcome to the Personal Information Manager
      </h2>
      <p
        className="lead"
      >
        This page helps to keep your personal information organized in a practical way.
      </p>
    </div>
  </header>
  <section
    id="notifications"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-8 mx-auto"
        >
          <h2>
            Notfications
          </h2>
          <kbd>
            Wed Oct 18 2017
          </kbd>
          <ul />
        </div>
      </div>
    </div>
  </section>
</div>
`;
